src/containers/ProjectContentsContainer.jsx

```diff
@@ -18,20 +21,38 @@ class ProjectContentsContainer extends Component {
     this.handleClick = this.handleClick.bind(this);
     this.state = {
       modalOpen: false,
-      translationText: ''
+      projectContents: {
+        description: '',
+        introduction: '',
+        title: '',
+      },
+      translationText: '',
     };
   }
 
   componentDidMount() {
-    const { actions } = this.props;
-    return actions.fetchProjectContents(this.props.params.project_id);
+    const { actions, params } = this.props;
+    return actions.fetchProjectContents(params.project_id);
   }
 
   handleClick(event) {
-    this.setState({
-      modalOpen: true,
-      translationText: event.target.textContent
-    });
+    if (isElementTranslatable(event)) {
+      const projectContentsData = (event.target.dataset.translationProjectContents)
+      ? event.target.dataset.translationProjectContents
+      : null;
+
+      this.setState({
+        modalOpen: true,
+        projectContents: {
+          description: projectContentsData.description,
```

Comment:
Is this just copying `projectContentsData` to `projectContents`? If so, why not rename `projectContentsData` to `projectContents`?